### DISCUSSION TITLE
Issue: Live Timing stops working

### DISCUSSION THREAD

**Discussion by Turrican76** [f1multiviewer/issue-tracker#9] [General]
It seems that the live timing is loosing connection every view minutes. In Cockpit View the Timing isn`t updated and can only be fixed by opening the live timing again and allowing it to resync.

**Comment by jeppeolesen** [comment]
Hey @Turrican76. Closing this discussion as stale. If it's still relevant in new versions of MultiViewer please submit an issue: https://github.com/f1multiviewer/issue-tracker/issues
